zend_abstract_interface/symbols/call.c

```diff
@@ -96,40 +341,61 @@ bool zai_symbol_call_impl(
         }
     }
 
-    volatile int zai_symbol_call_result = FAILURE;
+    // clang-format off
+    volatile int  zai_symbol_call_result    = FAILURE;
+    volatile bool zai_symbol_call_exception = false;
+    volatile bool zai_symbol_call_bailed    = false;
 
     zai_sandbox sandbox;
     zai_sandbox_open(&sandbox);
 
+    zai_symbol_call_arena_t arena;
+    zend_try {
+        zai_symbol_call_arena_create(&arena, &fcc, argc);
+    } zend_catch {
+        zai_symbol_call_bailed = true;
+    } zend_end_try();
+
+    if (zai_symbol_call_bailed) {
+        zai_sandbox_bailout(&sandbox);
+        zai_sandbox_close(&sandbox);
+        return false;
```

Comment:
Nit: zai_symbol_call_bailed var is redundant, can be moved directly into the catch block